notifications_api_common/exponential_backoff.py

```diff
@@ -0,0 +1,15 @@
+import random
+
+
+def get_exponential_backoff_interval(
+    factor: int, retries: int, maximum: int, base: int, full_jitter: bool = False
+) -> int:
+    """Calculate the exponential backoff wait time."""
```

Comment:
Could you also mention that this was taken from celery and overridden to make sure the base factor is configurable?